### PR TITLE
Korean translation updated

### PR DESCRIPTION
**Pull request by Psycool3695**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information: Added and improved missing Korean strings and fixed sumava map text duplication
    

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: It will take effect if you set the language to Korean and play. However, this does not apply to dedicated servers because they only support English.

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/maps/Stringtable.xml [hunk 1]
```diff
@@ -208,14 +209,14 @@
                 <German>Antistasi Ultimate - Rote Ernte</German>
                 <Russian>Antistasi Ultimate - Красная жатва</Russian>
                 <Chinesesimp>Antistasi Ultimate - 蓝色收获</Chinesesimp>
-                <Korean>안티스타시 플러스 - 빨갱이 수확</Korean>
+                <Korean>안티스타시 플러스 - 하베스트 레드</Korean>
```
Comment (Silence-112) [inline A3A/addons/maps/Stringtable.xml]: Does this read Antistasi Plus - Harvest Red?

When we changed the names of the maps, we didn't know what Ultimate would be in Korean.
Not a big deal if you can't change it, just wanted to give a heads up about it

Comment (wersal454) [reply A3A/addons/maps/Stringtable.xml]: 안티스타시 얼티밋 - 하베스트 레드 ? Dunno, google translated it

Comment (Psycool3695) [reply A3A/addons/maps/Stringtable.xml]: Oops, my mistake. I didn't realize I had switched to Ultimate. I'll fix it soon.

Comment (Silence-112) [reply A3A/addons/maps/Stringtable.xml]: Thanks

**Review by Silence-112 — COMMENTED** [review]
Generally looks fine to me, will test at some point

**Comment by Silence-112** [comment]
No errors on my side, everything appears to be working
I can go ahead and merge if you have nothing else to change

**Pull request merged**